server/spec/healing_spec.rb

```diff
@@ -188,19 +188,11 @@
     consumer = cp_user.register("foofy_test", :system, nil,
       {'cpu.cpu_socket(s)' => '8'}, nil, owner['key'], [], [])
     consumer_cp = Candlepin.new(nil, nil, consumer.idCert.cert, consumer.idCert['key'])
-    exception_thrown = false
 
-    begin
-      consumer_cp.consume_product()
-    rescue RestClient::BadRequest => e
-      exception_thrown = true
-      ex_message = "Ignoring request to auto-attach. " +
-        "It is disabled for org \"#{owner['key']}\" because of the content access mode setting."
-      data = JSON.parse(e.response)
-      expect(data['displayMessage']).to eq(ex_message)
-    end
+    consumer_cp.consume_product()
```

Comment:
This spec test is identical with the other one in `autobind_disabled_for_owner_spec.rb `. Let's delete it.

Reply:
Done.